### PR TITLE
Installer Warnings: better alert messages

### PR DESCRIPTION
**Pull request by 810**
Test instructions:

Go to the warnings view on the manage page. Instead of an info alert, its now an success alert, when there are no warnings.

Also if there are warnings, you have now an warning alert.

### CODE REVIEW

**Comment by infograf768** [comment]
How can we create a Warning, to test the new message?

**Comment by 810** [comment]
what I did,change:

administrator\components\com_installer\models\warnings.php line 113 to
````		if ($memory_limit > 8) ````

**Comment by 810** [comment]
@joomla-cms-bot please re-run the CI

**Comment by mbabker** [comment]
Current staging branch needs to be merged into the branch your PR is based on.  Travis runs based on a branch, it doesn't try merging the pull request's patch into the integration branch.

**Comment by 810** [comment]
Branch updated.

**Comment by andrepereiradasilva** [comment]
Just a suggestion, why not, instead of hardcoded Html for the message container, use the `enqueueMessage` method in the `view.html.php` file like in other views?

For instance https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_installer/views/update/view.html.php#L67

**Comment by mikeveeckmans** [comment]
I have tested this item :white_check_mark: successfully on beac6a224199a76824b507301d39d6f1b31ee667<br /><br />TEST OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on beac6a224199a76824b507301d39d6f1b31ee667<br /><br />This PR uses alert-warning but we dont have a class alert-warning in the css<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Comment by brianteeman** [comment]
When there are NO warning detected 
Before the patch the class is Alert-info 
After the patch the class is Alert-success

I dont see why this was changed - no action has been performed so it cant be a success ;)
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Comment by 810** [comment]
we need to standardize the colors.

update and discover = yellow (alert) = ok
standard and database = blue (info) = ok

**Comment by andrepereiradasilva** [comment]
why no use `enqueueMessage`, example:
```php
<?php if (!count($this->messages)) : ?>
	<?php JFactory::getApplication()->enqueueMessage(JText::_('COM_INSTALLER_MSG_WARNINGS_NONE'), 'notice'); ?>
<?php else : ?>
```

**Comment by andrepereiradasilva** [comment]
Not wanting to be picky as, as it is, it's already and improvement, but shouldn't that messages be outside the template?

Maybe that check and the message should be in the `view.html.php` file:
https://github.com/810/joomla-cms/blob/warning/administrator/components/com_installer/views/warnings/view.html.php#L34

Other views do this, check the `com_installer` `update` view for instance
https://github.com/810/joomla-cms/blob/warning/administrator/components/com_installer/views/update/view.html.php#L65-L68

**Comment by 810** [comment]
@andrepereiradasilva updated

File: administrator/components/com_installer/views/warnings/view.html.php [hunk 1]
```diff
@@ -32,6 +32,15 @@ public function display($tpl = null)
 		$items = $this->get('Items');
 		$this->messages = &$items;
 		parent::display($tpl);
+
+		if (count($items) > 0)
+		{
+			JFactory::getApplication()->enqueueMessage(JText::_('COM_INSTALLER_MSG_WARNINGS_NOTICE'), '');
```
Comment (andrepereiradasilva) [inline administrator/components/com_installer/views/warnings/view.html.php]: shouldn't be 'warning' here?
the rest i tested ok.

Comment (810) [inline administrator/components/com_installer/views/warnings/view.html.php]: updated

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 2d36d1df45866ea8d82e62d3dfc02108877d9cb2<br /><br />thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Comment by mikeveeckmans** [comment]
I have tested this item :white_check_mark: successfully on 2d36d1df45866ea8d82e62d3dfc02108877d9cb2<br /><br />TEST OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Comment by brianteeman** [comment]
RTc -thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8780">issues.joomla.org/joomla-cms/8780</a>.</sub>

**Pull request merged**